### ISSUE TITLE
Only send one notification unless `--wait` is used.

### ISSUE THREAD

**Issue by dtuite** [larderdev/kubewise#45]
When helm is used without `--wait`, it's basically impossible for the install or upgrade to fail. Helm doesn't wait for the pods to start up or anything, it just returns to the user immediately. There is no information available in the second notification that couldn't have been sent in the first.

When `--wait` is used, it makes sense to send two notifications because the upgrade or install can fail and the second notification can indicate that to the user.

As far as I can tell, there is no way to determine if `--wait` was used or not from within the cluster. Might need to talk to the Helm owners.

**Comment by hd-deman** [comment]
@dtuite How about global way to disable wait behaviour to reduce noice?

**Comment by hd-deman** [comment]
And/or set max retries?

**Comment by dtuite** [comment]
> @dtuite How about global way to disable wait behaviour to reduce noice?

Yes. This is another option. I presume that most teams will have scripted their helm deployments and either will always be using `--wait` or never using `--wait`. There shouldn't be many teams using a mix outside of dev clusters.

**Comment by TJM** [comment]
I would like to see this as well, the extra "Upgraded" notification is not very useful to us. We are using helm-operator, and generally have to watch its logs or `k describe helmrelease xxxxxx` to see whats happening :)
